cypress/integration/data-import/edi-import-file-for-create-invoice-record.spec.js

```diff
@@ -24,7 +25,13 @@ describe('ui-data-import: EDIFACT file import with creating of new invoice recor
     );
   });
 
-  it('C345423 EDIFACT file import with creating of new invoice record', { tags: [testTypes.smoke] }, () => {
+  afterEach(() => {
+    const invoiceNumberFromEdifactFile = '94999';
```

Comment:
may be this constant should locate somewhere in fragments

Reply:
Done